src/html5sortable.ts

```diff
@@ -428,7 +428,7 @@ export default function sortable (sortableElements, options) {
       dragging.style.display = dragging.oldDisplay
       delete dragging.oldDisplay
 
-      placeholders.forEach(_detach)
+      placeholderMap.forEach(_detach)
```

Comment:
This is wrong.. it should be Array.from(placeholderMap.values()).forEach(_detach);

Reply:
I thought that too, but `placeholderMap.forEach` should call `_detach` with the following argument according to mdn:

- the element value
- the element key
- the Map object being traversed

since the value is the element, it should actually work like this `placeholderMap.forEach(_detach)` right?

Reply:
I agree it should but it doesn't look like it. Changing to Array.from(placeholderMap.values()).forEach(_detach) works.

Reply:
Okay, weird.